addons/web/static/tests/legacy/fields/relational_fields/field_one2many_tests.js

```diff
@@ -8536,8 +8536,10 @@ QUnit.module('Legacy fields', {}, function () {
 
             // write something in the field
             var $input = form.$('input[name="turtle_foo"]');
-            await testUtils.fields.editInput($input, value);
-            await testUtils.fields.triggerKeydown($input, 'enter');
+            $input[0].value = value;
+            await testUtils.dom.triggerEvent($input, "input");
+            testUtils.fields.triggerKeydown($input, 'enter');
```

Comment:
onchange has not been launched --> we should launch it and take decision on editedRecord validity when onchange has finished